scr/script.js

```diff
@@ -0,0 +1,105 @@
+if ('library' in localStorage && localStorage.getItem('library') === '') {
+  const myLibrary = [];
+  const i = 0;
+  localStorage.setItem('counter', i);
+  localStorage.setItem('library', myLibrary);
+}
+
+function Book(name, author, pages, read, num) {
+  this.name = name;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+  this.num = num;
+}
```

Comment:
### ES6 Destructuring Syntax
I most encourage you to use ES6 Syntax all around your project, now I'm not going to repeat this issue on every function instead I will leave you here an example of how to use it.

```Javascript
const Book = (name,author,page,read,num) => {
  return(
    {
      name,
      author,
      page,
      read,
      num
    }
  )
}
//
// NOTICE THAT NOW YOU CAN CALL EVERYTHING BY DOING AS FOLLOWING
//
const {name, author, page, read, num} = Book('test','me','123',true,'1')
// OR 
const book = Book('test','me','123',true,'1')
const {name, author, page, read, num} = book
//
// AND HERE IS HOW YOU CAN TEST IT WORK
//
console.log(`Name = ${name}`);
console.log(`Author = ${author}`);
console.log(`Page = ${page}`);
console.log(`Read = ${read}`);
console.log(`Num = ${num}`);
```
If you have any question of how to use ES6 destructuring I recommend you to read [this doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)